tests/utils.go

```diff
@@ -4584,6 +4631,11 @@ func IsRunningOnKindInfra() bool {
 	return strings.HasPrefix(provider, "kind")
 }
 
+func IsRunningOnKindInfraIPv6() bool {
```

Comment:
The issue your PR is trying to fix happens in all kind based providers, as stated in the [PR's description](https://github.com/kubevirt/kubevirt/pull/3374#issue-413443532).

As such, I don't think this function is required at all, and the `IsRunningOnKindInfra` function should be used instead.

Reply:
as explained in the previous commit,
if will break the other kind providers, since they don't have the file,
failure to mount, result in failure to run the launcher.

i will chance the PR description as Edy asked

Reply:
Obviously. That's is why I've stated I'm [my review comment](https://github.com/kubevirt/kubevirt/pull/3374#pullrequestreview-405725579) what I think should be done (i.e. generate the product id in the kind common script).

I'm OK with it being done in a follow-up PR.